packages/calypso-build/webpack.config.js

```diff
@@ -112,6 +112,12 @@ function getWebpackConfig(
 					presets,
 					workerCount,
 				} ),
+				TranspileConfig.loader( {
+					cacheDirectory: true,
+					include: shouldTranspileDependency,
```

Comment:
We'll want to expose this to the user of calypso-build, as they may have dependencies that require transpilation that calypso does not.

Reply:
Do users need to only expand the whitelist with new entries, or to also exclude some existing ones?